tests/devices/default_qubit/test_default_qubit_plxpr.py

```diff
@@ -23,6 +23,7 @@
 
 @pytest.fixture(autouse=True)
 def enable_disable_plxpr():
+    """Enable and disable plxpr."""
     qml.capture.enable()
     yield
     qml.capture.disable()
```

Comment:
We don't need this fixture anymore. You can just update the `pytestmark` above to be:
```python
pytestmark = [pytest.mark.jax, pytest.mark.usefixture("enable_disable_plxpr")]
```

Reply:
updated.